x/staking/keeper/delegation.go

```diff
@@ -577,12 +577,12 @@ func (k Keeper) Unbond(
 
 	isValidatorOperator := delegation.DelegatorAddress.Equals(validator.OperatorAddress)
 
-	// if the delegation is the operator of the validator and undelegating will decrease the validator's self delegation below their minimum
-	// trigger a jail validator
+	// If the delegation is the operator of the validator and undelegating will decrease the validator's
+	// self-delegation below their minimum, we jail the validator.
 	if isValidatorOperator && !validator.Jailed &&
```

Comment:
Note there is no change here apart from keeping the code DRY and calling `k.Jail` instead of manually calling the private internal APIs.